README.md

```diff
@@ -18,17 +18,11 @@ kube-applier serves a [status page](#status-ui) and provides [metrics](#metrics)
 * [Go (1.7+)](https://golang.org/dl/)
 * [Docker (17.05+)](https://docs.docker.com/engine/getstarted/step_one/#step-1-get-docker)
 * [Kubernetes cluster](http://kubernetes.io/docs/getting-started-guides/binary_release/)
+    * kube-applier generally supports any Kubernetes server release, assuming that you are installing a compatible kubectl client in your Dockerfile.
     * The kubectl version specified in the Dockerfile must be either the same minor release as the cluster API server, or one release behind the server (e.g. client 1.3 and server 1.4 is fine, but client 1.4 and server 1.3 is not).
-    * Supported Kubernetes releases:
-        * Note: Releases prior to 1.6.0 are subject [many](https://github.com/kubernetes/kubernetes/issues/40119) [known](https://github.com/kubernetes/kubernetes/issues/29542) [issues](https://github.com/kubernetes/kubernetes/issues/39906) with using `kubectl apply` to apply ThirdPartyResource objects.
-        * 1.2.x
-        * 1.3.x
-        * 1.4.x
-        * 1.5.8+ (earlier 1.5 releases not supported due to [issue](https://github.com/kubernetes/kubernetes/issues/7789#issuecomment-280568960), fixed [here](https://github.com/kubernetes/kubernetes/pull/44862/)).
-        * 1.6.3+ (same as 1.5)
-        * 1.7.x
-        * 1.8.x
-        * 1.9.x
+    * There are several known problems with "kubectl apply" that may affect your use of kube-applier. Some examples:
```

Comment:
"kubectl apply" -> `kubectl apply`?
Just to be consistent with the same content below.